src/Accounts/AccountsView.js

```diff
@@ -13,22 +15,30 @@ const printKeyValue = (label, val, isRequire) => {
   );
 };
 
-const renderAccount = (val, key) => (
-  <Row
-    className={css.account}
-    key={key}
-  >
-    {printKeyValue('name', get(val, ['name'], ''), false)}
-    {printKeyValue('accountNumber', get(val, ['accountNo'], ''), false)}
-    {printKeyValue('description', get(val, ['description'], ''), false)}
-    {printKeyValue('payable', get(val, ['appSystemNo'], ''), false)}
-    {printKeyValue('paymentMethod', get(val, ['paymentMethod'], ''), false)}
-    {printKeyValue('account.contactInfo', get(val, ['contactInfo'], ''), false)}
-    {printKeyValue('libraryCode', get(val, ['libraryCode'], ''), false)}
-    {printKeyValue('libraryEDICode', get(val, ['libraryEdiCode'], ''), false)}
-    {printKeyValue('notes', get(val, ['notes'], ''), false)}
-  </Row>
-);
+const renderAccount = (val, key) => {
+  const paymentMethod = get(val, 'paymentMethod');
+
+  return (
+    <Row
+      className={css.account}
+      key={key}
+    >
+      {printKeyValue('name', get(val, ['name'], ''), false)}
```

Comment:
I don't think we need to use `printKeyValue` at all

Reply:
Yes. I know. I added `PaymentMethod` only and didn't touch the rest code.

Reply:
well, it's green in github and marked as "touched") I see 2 options - change only lines you need or refactor to not use `printKeyValue`

Reply:
Done